package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/root-config-lib.js",
   "scripts": {
     "clean": "rimraf dist",
-    "start": "webpack-dev-server --port",
+    "start": "webpack-dev-server",
```

Comment:
The convention established by fatma and me is that to run the project you should do `npm start -- --port 8081`